scalafmt-core/shared/src/main/scala/org/scalafmt/config/Docstrings.scala

```diff
@@ -17,6 +19,7 @@ import metaconfig._
   */
 case class Docstrings(
     oneline: Docstrings.Oneline = Docstrings.Oneline.keep,
+    wrap: Boolean = false,
```

Comment:
My experience is that Boolean config options usually end up becoming enums. I think it’s worth considering making this option an enum instead.

Reply:
yep... i was thinking the same, as i had just added the `Comments.wrap` parameter as an enum. good catch.